bin/misc-functions.sh

```diff
@@ -259,7 +259,9 @@ install_qa_check() {
 	# Prematurely delete WORKDIR in case merge-wait is enabled to
 	# decrease the space used by portage build directories until the
 	# packages are merged and cleaned.
-	if has merge-wait ${FEATURES} && ! has keepwork ${FEATURES}; then
+	if has merge-wait ${FEATURES} && \
+		! has keepwork ${FEATURES} && \
```

Comment:
The newline escapes are unnecessary when `&&` appears as the last word on the line.